templates/python-package-to-windows-webapp-on-azure.yml

```diff
@@ -0,0 +1,101 @@
+# Python package to Windows Web App Azure
+# Build your Python project and deploy it to Azure as a Windows Web App.
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+- master
+
+variables:
+  # Azure Resource Manager connection created during pipeline creation
+  azureServiceConnectionId: '{{ azureServiceConnection.Id }}'
+  
+  # Web app name
+  webAppName: '{{ webAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'windows-latest'
+
+  # Environment name
+  environmentName: '{{ webAppName }}'
+
+  # Project root folder. Point to the folder containing manage.py file.
+  projectRoot: $(System.DefaultWorkingDirectory)
+  
+  # Name of the folder containing 'static' folder
+  appFolderName: "app"
+
+stages:
+- stage: Build
+  displayName: Build stage
+  jobs:
+  - job: BuildJob
+    pool:
+      vmImage: $(vmImageName)
+    steps:
+    - task: ArchiveFiles@2
+      displayName: 'Archive files'
+      inputs:
+        rootFolderOrFile: '$(projectRoot)'
+        includeRootFolder: false
+        archiveType: zip
+        archiveFile: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
+        replaceExistingArchive: true
+    
+    - upload: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
+      displayName: 'Upload package'
+      artifact: drop
+
+- stage: Deploy
+  displayName: 'Deploy Web App'
+  dependsOn: Build
+  condition: succeeded()
+  jobs:
+  - deployment: DeploymentJob
+    pool:
+      vmImage: $(vmImageName)
+    environment: $(environmentName)
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+          - task: AzureAppServiceManage@0
+            displayName: 'Install Python Extension'
+            inputs:
+              azureSubscription: $(azureServiceConnectionId)
+              Action: 'Install Extensions'
+              WebAppName: $(webAppName)
+              ExtensionsList: python353x86
+              OutputVariable: 'PYTHON_EXT'
```

Comment:
this input is disable I gues,, can u check this variable? `ExtensionsList: '$(D)`

Reply:
As discussed offline. Continuing to use it.